### DISCUSSION TITLE
Pb between c++ and python

### DISCUSSION THREAD

**Discussion by BenLag2906** [apache/incubator-mxnet#10529] [Q&A]
Hi,

I have a problem my train is done in python, and when i try to use my json file and params in c++.

It crashed at MXPredSetInput althought it's ok with other Json file like SSD.

My Json file : 

  "nodes": [
    {
      "op": "null", 
      "name": "data", 
      "inputs": []
    }, 
    {
      "op": "_copy", 
      "name": "id", 
      "inputs": [[0, 0, 0]]
    }, 
    {
      "op": "null", 
      "name": "bn_data_gamma", 
      "attr": {
        "eps": "2e-05", 
        "fix_gamma": "True", 
        "momentum": "0.9"
      }, 
      "inputs": []
    }, 

Could you please give a piece of advise ?

Regards

**Comment by ThomasDelteil** [comment]
@BenLag2906 it looks like your network didn't export properly, can you put the code of how you exported this network ? Thanks!

**Comment by lupesko** [comment]
@indhub can you please remove the label "Question" and add "Bug"? this seems like a bug report, not a question.

**Comment by indhub** [comment]
@lupesko There isn't enough information to consider this a bug. That JSON file doesn't look normal. I would wait for the user to provide more information about the model and how it was exported.

**Comment by BenLag2906** [comment]
Hi,
I give you my entire json file . In c++ it crashed, in python it's OK to launch CNN.

[yks_pose-symbol.zip](https://github.com/apache/incubator-mxnet/files/2083582/yks_pose-symbol.zip)

regards

`

**Comment by BenLag2906** [comment]
Hi,
For adding a precision, I use the same deploy script as SSD. Is there another way deploy it ?

regards.

**Comment by BenLag2906** [comment]
To add more explanation, i try to translate kohillyang code in c++, but many difficulties persist.

**Comment by leleamol** [comment]
@mxnet-label-bot update [Question, C API]
